app/scripts/lib/cropper.js

```diff
@@ -24,6 +24,10 @@ function () {
    * }
    */
   function Cropper (options) {
+    function isNotUndefined (value, defaultValue) {
+      return typeof value !== 'undefined' ? value : defaultValue;
```

Comment:
We could also just use underscore.js' [`_.isUndefined()`](http://underscorejs.org/#isUndefined) here. Something like:
```js
return !_.isUndefined(value) ? value : defaultValue;
```
(or remove the `!` and reverse the argument order, whatever is easier to read+understand)

---

There is also a [`_.result()`](http://underscorejs.org/#result) method, but I'm not sure if it could help us here since it operates on an object. ... Unless we could do something like:

```js
this.displayLength = _.result(options, 'displayLength', DEFAULT_DISPLAY_LENGTH);
```